ui/src/overridden/components/Menu/Logo.js

```diff
@@ -14,7 +14,7 @@ export const Logo = ({ ...props }) => {
 export const LogoMobile = ({ ...props }) => {
   return (
     <h3>
-      <Link to="/" className="logo">
+      <Link to="/" className="logo size">
```

Comment:
I think class name size is used in SUI, I would suggest to use something like `.media-font-size`